lib/data_update_scripts/20211209213849_remove_dangling_user_reactions.rb

```diff
@@ -0,0 +1,11 @@
+module DataUpdateScripts
+  class RemoveDanglingUserReactions
+    def run
+      # reactions to users who have been deleted should be removed:
+      Reaction
+        .where(reactable_type: "User")
+        .where.not(reactable_id: User.all.select(:id))
+        .delete_all
```

Comment:
I would be more inclined for disciplines sake to use `find_each` or a batches approach, but the count in DEV (where I expect the count to be highest) is 27.

However - we _also_ have 1000+ comment reactions that are dangling and I'll look into that next.